src/render/DrawContext.js

```diff
@@ -419,6 +419,13 @@ define([
 
             // Intentionally not documented.
             this.pixelSizeOffset = 0;
+
+            /**
+             * A place to store gl extensions for this drawContext.
+             * @type {Object}
+             * @private
+             */
+            this._glExtensionsCache = Object.create(null);
```

Comment:
This notation with `_` and `@private` is new to this class. For consistency, I would use the same approach as above for `pixelSizeOffset`. Or is this the new convention @pdavidc ?